Gemfile.lock

```diff
@@ -226,6 +228,7 @@ DEPENDENCIES
   activerecord-session_store
   cancan
   combine_pdf (~> 0.2.12)
+  composite_primary_keys (~> 8.0)
```

Comment:
To jsme si rekli ze nebudeme pouzivat, ne?

Reply:
promiň, já myslel, že už jsem to smazal. Každopádně nikde v kódu už to nepoužívám, takže to nebude problém